<header class="page-header">
    <div class="container-fluid">
       <h2 class="no-margin-bottom">ProteomiX from data to knowledge</h2>
    </div>
</header>
<section class=" no-padding-bottom">
<div class="container-fluid">
    <mat-card>
        <a href="#">
            <img height="250" width="600" id="logo_proteomix" src="assets/images/logo_proteomix.png" >
        </a> 
        <a href="https://cgfb.u-bordeaux.fr/">
            <img height="250" width="600" id="logo_cgfb" src="assets/images/cbib.jpg" >
        </a> 
    </mat-card>
    <br>
    <mat-card>
        <div>
            <p style="font-size: 12pt"> 
                
                            Study of protein expression data, in particular relative quantification data in label-free proteomics obtained by high-speed mass spectrometry, has become widely available and democratized within the scientific community, thanks in particular to boom in this discipline in recent years. This type of data today allows access to precise mapping of the biological processes involved in a cell, a cell type, an organism or a population. High-throughput proteomics is now widely used for protein expression profiling, but the bioinformatics analysis of resultant data can be time-consuming and challenging, especially for biologists. We aim to streamline the bioinformatic analyses of protein-level data by developing a user-friendly, interactive web application for exploratory data analysis and visualization, differential expression, enrichment and pathway analysis.
        </div>
    </mat-card>
    <br>
    <mat-card>
        <h3>Project description</h3>
        <p style="font-size: 12pt">
        </p>
        <p style="font-size: 12pt">
            To meet these needs, we have developed an easy-to-use web application ProteomiX designed to helps biologists easily process and explore Proteomics data, for in-depth analysis of protein expression data. ProteomX encapsulates useful R and python packages, annotation databases, and related web services. The input is a protein-level expression matrix obtained from Proteome Discoverer 2, Progenesis, MaxQuant, or other platforms. Main functionalities include (1) seamless guidance to the user to fill in relevant meta-data, (1) data pre-processing, (2) exploratory data analysis, (3) differential expression, (4) enrichment, (5) pathway analysis and visualization (6) data integration.
        </p>
    </mat-card>
    <br>

    <mat-card>
        <h3>Start experiment ProteomX</h3>
        <button class="btn btn-outline-rounded btn-info" type="submit" (click)="onSelect()"> Start analysis
                    <span style="margin-left:10px;" class="glyphicon glyphicon-arrow-right"></span>
                </button><span style="margin-right:10px;"></span>            
    </mat-card>
</div>
</section>